cli/src/config/misc.toml

```diff
@@ -5,6 +5,8 @@ amend = ["squash"]
 co = ["checkout"]
 unamend = ["unsquash"]
 
+[revsets]
```

Comment:
We have talked about merging `[revsets]` and `[revset-aliases]` so it's probably best to use a different section for things that are not revsets

Reply:
I actually put it under `core.` at first, but then put it under `revsets.` since it seemed more "topical." Do you think `core.` is a better namespace?

Reply:
`ui.` works for me if we don't have a better place. it's specific to the UI, but so are almost all configs (`user.name` and `user.email` being good examples of non-UI configs)

Reply:
Migrated, thanks